modules/dbnd/src/dbnd/conf/databand-core.cfg

```diff
@@ -134,6 +134,7 @@ validate_no_extra_params = error
 heartbeat_timeout_s = 900
 heartbeat_interval_s = 5
 heartbeat_sender_log_to_file = True
+open_web_tracker_in_browser = False
```

Comment:
Remove from here, set default to `False` in `RunConfig` class

Reply:
ok, done